tools/modules/software/module_immich.sh

```
@@ -0,0 +1,152 @@
+	["module_immich,status"]="Active"
+	["module_immich,doc_link"]="https://immich.app/docs"
+	["module_immich,group"]="Media"
+	["module_immich,port"]="8077"
+	["module_immich,arch"]="x86-64 arm64"
+)
+#
+# Module immich
+#
+function module_immich () {
+	local title="immich"
+	local condition=$(which "$title" 2>/dev/null)
```

Shellcheck (0.10.0)

[warning] condition appears unused. Verify use (or export if used externally).

(SC2034)